<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Student</class>
 <widget class="QMainWindow" name="Student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Account information:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_signin">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign in</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_unknown">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_idconst">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_balanceconst">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Balance:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_id">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_balance">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>439</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
